packages/core/src/util/txInspector.ts

```diff
@@ -17,47 +17,49 @@ type TxInspector<T extends Inspectors> = (tx: TxAlonzo) => {
   [k in keyof T]: ReturnType<T[k]>;
 };
 
-// Inspectors result types
+// Inspection result types
```

Comment:
Please separate the isolated refactor within this util into a separate commit

Reply:
Also, changes in `wallet` package are breaking. We're using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) to indicate it